engine/include/cubos/engine/input/axis.hpp

```diff
@@ -74,11 +75,20 @@ namespace cubos::engine
         /// @param value New value.
         void value(float value);
 
+        /// @brief Gets the deadzone.
+        /// @return Deadzone.
+        float deadzone() const;
+
+        /// @brief Sets the deadzone.
+        /// @param deadzone New deadzone.
+        void setDeadzone(float deadzone);
+
     private:
         std::vector<InputCombination> mPositive;
         std::vector<InputCombination> mNegative;
         std::vector<core::io::GamepadAxis> mGamepadAxes;
 
         float mValue{0.0F}; ///< Not serialized.
+        float mDeadzone{0.0F};
```

Comment:
You also need to add this field in the reflection definition for the `InputAxis`, otherwise it won't get picked up when reading the bindings from a file (check out the CUBOS_REFLECT_IMPL in axis.cpp)